apps/src/templates/sectionProgressV2/progress-table-v2.module.scss

```diff
@@ -227,14 +227,15 @@ $level-cell-width: 52;
 
 .sortDropdown {
   height: 76px;
-  overflow-x: hidden;
+  width: fit-content;
```

Comment:
@levadadenys I noticed that this was required to not have the caret floating in space next to the div. If width isn't variable, maybe we should add this to the design component itself.

Reply:
Interesting, haven't encounetered this issue in storybook. Are you sure it's not use case specific thing?
If it can be used anywhere I think we can update SimpleDropdown component styles in a follow up PR (also updating changelog.md)